bin/webpack.js

```diff
@@ -166,9 +166,11 @@ if(argv.init) {
 	if (!filePaths.length) {
 		throw new Error('Please specify a path to your webpack config');
 	}
+	const outputConfigName = filePaths[1] || `${filePaths[0]}v2.js`;
```

Comment:
At the least we should store it somewhere, but only as a temp trial for now, or if user specifies.